### PR TITLE
Additional sql->comdb2 type mappings

### PR DESCRIPTION
**Pull request by nirbhayc**

### CODE REVIEW

File: sqlite/comdb2build.c [hunk 1]
```diff
@@ -1913,9 +1921,23 @@ static void free_ddl_context(Parse *pParse)
...
+    size_t type_len;
+    int size_maybe;
+
+    type_len = strlen(type);
 
     for (int i = 0;
          i < sizeof(type_mapping) / sizeof(struct comdb2_type_mapping); ++i) {
+
+        /* Check if current type could accept size. */
+        int size_maybe =
```
Comment (riverszhang89) [inline sqlite/comdb2build.c]: You already defined size_maybe above.

Comment (riverszhang89) [inline sqlite/comdb2build.c]: size_maybe = (type_mapping[i].flag & COMDB2_TYPE_FLAG_ALLOW_ARRAY);

Comment (riverszhang89) [inline sqlite/comdb2build.c]: Also the name "_maybe_" sounds confusing. Change it to something like "_accepts_size_"?

**Comment by nirbhayc** [comment]
Thanks @riverszhang89 !

**Pull request merged**